myslam:
  ros__parameters:

      # ROS parameters
      odom_frame: odom
      map_frame: map
      base_frame: base_footprint
      scan_topic: /lidar

      throttle_scans: 1
      transform_timeout: 0.2
      minimum_time_interval: 0.5

      # Mapper parameters
      minimum_travel_distance: 0.5
      minimum_travel_heading: 0.5

      
    